proxy_idp_auth_test_oidc.sh

```diff
@@ -7,6 +7,13 @@
 DIR="${0%/*}"
 source "${DIR}/proxy_idp_auth_test_config.sh"
 
+CACHE_LAST_MODIFIED=$(($(date +%s) - $(date +%s -r ${DIR}/proxy_idp_auth_test_oidc.cache)))
+
+if [ ${CACHE_LAST_MODIFIED} -le ${CACHE_TIME} ]; then
+  echo "$(<${DIR}/proxy_idp_auth_test_oidc.cache)"
```

Comment:
It should be good to check cached data:
* Only one line in the file
* Base structure by regex